docs/en/setup/advanced/MeterReporter.md

```diff
@@ -0,0 +1,110 @@
+# Python Agent Meter Reporter
+
+To enable or disable this feature, you will need to set some environment variables.
+
+
+## Enabling the feature (default)
+```Python 
+os.environ['SW_AGENT_METER_REPORTER_ACTIVE'] = 'True'
+``` 
+or
+```bash
+export SW_AGENT_METER_REPORTER_ACTIVE=True
+```
+
+## Disable the feature
+```Python 
+os.environ['SW_AGENT_METER_REPORTER_ACTIVE'] = 'False'
+``` 
+or
+```bash
+export SW_AGENT_METER_REPORTER_ACTIVE=False
+```
+## Counter
+* `Counter` API represents a single monotonically increasing counter, automatic collect data and report to backend.
+```python
+tg_ls = [MeterTag("key", "value")]
```

Comment:
Would be awesome if we can just pass in a Python tuple

```suggestion
tg_ls = [("key", "value")]
```

Reply:
Sounds like a good idea.